learn/sherif/main_test/src/main/java/frc/robot/subsystems/IntakeControl.java

```diff
@@ -0,0 +1,34 @@
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj.PWMTalonSRX;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+public class IntakeControl extends SubsystemBase {
+    private final PWMTalonSRX m_throat_left;
+    private final PWMTalonSRX m_throat_right;
+    private final PWMTalonSRX m_elevator_left;
+    private final PWMTalonSRX m_elevator_right;
```

Comment:
Make sure you also include the outer intake wheel, and change to elevator_front and elevator_back. Also, you should add solenoid code for both Intake and Shooter.

Reply:
Does the solenoid represent the piston? If so I would consider that a part of the shooter and would be part of a separate subsystem and PR.

Reply:
There are two solenoids in use. One controls the two air cylinders that change the shooter position (up/down). The other controls two more air cylinders, which extend and retract the outer intake mechanism. If you look at the picture posted to #general, the two air cylinders on the left (back of the robot) are for the shooter. The intake cylinders are not yet attached.

Reply:
We don't need to add solenoids yet.  Once we get the robot driving and shooting we can go back and add code for the solenoids.  Even before that we'll want to work on tuning the ball management to allow us to load as many balls as we can in the elevator without jamming.